lib/PHPCfg/Printer.php

```diff
@@ -121,16 +121,15 @@ protected function renderOp(Op $op)
 
         $result .= $this->renderAttributes($op->getAttributes());
 
-        if ($op instanceof Op\Stmt\Function_) {
+        if ($op instanceof Op\Stmt\Function_ || $op instanceof Op\Stmt\Class_ || $op instanceof Op\Stmt\Property || $op instanceof Op\Stmt\ClassMethod || $op instanceof Op\Expr\Param) {
```

Comment:
Yeah, this is getting a bit messy. I think it's fine for this PR (it isn't adding any complexity that isn't there), but maybe we should think on an Interface to check for to render the attribute groups... (or maybe just render all Stmts)?